README.md

```diff
@@ -29,9 +29,7 @@ This xsl converts `<ext-link>` elements within references (`<ref>`) that have a
 
 ### [/src/remove-supplementary-materials.xsl](/src/remove-supplementary-materials.xsl)
 
-This stylesheet is transforming an XML document by removing any "sec" element with a "sec-type" attribute value of "supplementary-material", and copying the remaining elements into a new "body" element in the output document.
-
-TODO: We need an expression of whether we believe this is work for biorXiv, encoda or EPP team.
+This stylesheet is transforming an XML document by removing any "sec" element with a "sec-type" attribute value of "supplementary-material". These supplementary material sections are only partially retained by encoda  - the files themselves need representation as downloadable files in the JSON. As it stands the labels and filepaths are decoded and reencoded as paragraphs. The current rendering on EPP as a result is less than ideal, where labels and filepaths are rendered in a series of paragraphs. The purpose of this xsl is to remove these sections until encoda and EPP can be updated to render downloadable supplementary files on the page. 
```

Comment:
If there is a ticker for Encoda, feel free to link to it here. Is there need for an EPP client piece until we have better handling in Encoda?

Reply:
I have found [the ticket for encoda](https://github.com/elifesciences/enhanced-preprints-issues/issues/116) and added to the readme. Presume we'd need work in EPP, but only after encoda fix is in. @QueenKraken can maybe confirm if a ticket already exists for the EPP side or if that remains un-raised prior to encoda fix.